osu.Framework/Graphics/Containers/BufferedContainer.cs

```diff
@@ -246,9 +246,10 @@ public bool RedrawOnScale
         /// <param name="formats">The render buffer formats attached to the frame buffers of this <see cref="BufferedContainer"/>.</param>
         /// <param name="pixelSnapping">Whether the frame buffer position should be snapped to the nearest pixel when blitting.
         /// This amounts to setting the texture filtering mode to "nearest".</param>
-        public BufferedContainer(RenderbufferInternalFormat[] formats = null, bool pixelSnapping = false)
+        /// <param name="clipToRootNode">Whether the frame buffer should be clipped to be contained in the root node.</param>
```

Comment:
```suggestion
        /// <param name="clipToRootNode">Whether the frame buffer should be clipped to be contained in the root node. Should only be disabled if you plan to draw to a region outside (or larger than) the screen.</param>
```

Might be better to clarify when this should be used, as it's a pretty special circumstance.